__docs__/wonder-blocks-dropdown/multi-select.stories.tsx

```diff
@@ -281,29 +284,30 @@ const ControlledMultiSelect = (args: PropsFor<typeof MultiSelect>) => {
     >(null);
     return (
         <View style={{gap: spacing.xSmall_8}}>
-            <MultiSelect
-                {...args}
-                id="multi-select"
-                opened={opened}
-                onToggle={setOpened}
-                selectedValues={selectedValues}
-                onChange={setSelectedValues}
-                validate={(values) => {
-                    if (values.includes("jupiter")) {
-                        return "Don't pick jupiter!";
-                    }
-                }}
-                onValidate={setErrorMessage}
-            >
-                {items}
-            </MultiSelect>
-            {(errorMessage || args.error) && (
-                <LabelMedium
-                    style={{color: semanticColor.status.critical.foreground}}
-                >
-                    {errorMessage || "Error from error prop"}
-                </LabelMedium>
-            )}
+            <LabeledField
+                label={label || "MultiSelect"}
+                errorMessage={
+                    errorMessage || (args.error && "Error from error prop")
+                }
+                field={
+                    <MultiSelect
+                        {...args}
+                        id="multi-select"
```

Comment:
**suggestion:** You could probably drop this `id` now that `LabeledField` handles it.... or maybe move it to `LabeledField`. One thing to consider is that we'd need to have unique IDs in these stories.

Reply:
Dropped the id since it's handled, good catch!